src/components/common/ScreenShotRow.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Box } from 'grommet';
+import ScreenshotCard from '../ScreenshotCard.js';
+
+const ScreenShotRow = ({
+  margin,
+  src0,
+  src1,
+  src2,
+  label0,
+  label1,
+  label2,
+  a11yTitle0,
+  a11yTitle1,
+  a11yTitle2,
```

Comment:
I'm thinking you can just use the label prop instead of `a11yTitle1`, it will clean up this props.